runtime/doc/lua.txt

```diff
@@ -9,24 +9,11 @@ Lua engine                                                           *lua* *Lua*
                                        Type |gO| to see the table of contents.
 
 ==============================================================================
-INTRODUCTION                                                       *lua-intro*
+INTRODUCTION
 
-The Lua 5.1 script engine is builtin and always available. Try this command to
-get an idea of what lurks beneath: >
-
-    :lua print(vim.inspect(package.loaded))
-
-Nvim includes a "standard library" |lua-stdlib| for Lua.  It complements the
-"editor stdlib" (|builtin-functions| and |Ex-commands|) and the |API|, all of
-which can be used from Lua code (|lua-vimscript| |vim.api|). Together these
-"namespaces" form the Nvim programming interface.
-
-The |:source| and |:runtime| commands can run Lua scripts. Lua modules can be
-loaded with `require('name')`, which by convention usually returns a table.
-See |lua-require| for how Nvim finds and loads Lua modules.
-
-See this page for more insight into Nvim Lua:
-    https://github.com/nanotee/nvim-lua-guide
+The Lua 5.1 script engine is builtin and always available. Nvim also includes
+a "standard library" |lua-stdlib| for Lua. See |lua-intro| for a quick
```

Comment:
I think we should keep a bit more; the whole paragraph about the "standard library" seems relevant.

(The rest can indeed be punted to `lua-intro`.)